_opensearch/query-dsl/full-text.md

```diff
@@ -402,26 +393,87 @@ GET _search
 }
 ```
 
+## Convert text with analyzers
+
+OpenSearch provides the analyzer option to convert your structured text into the format that works best for your searches. You can use the following options with the analyzer field: standard, simple, whitespace, stop, keyword, pattern, fingerprint and language. Different analyzers have different character filters, tokenizers, and token filters. The stop analyzer, for example, removes stop words (e.g. “an,” “but,” “this”) from the query string.
+
+OpenSearch supports the following language values with the `analyzer` option:
+arabic, armenian, basque, bengali, brazilian, bulgarian, catalan, czech, danish, dutch, english, estonian, finnish, french, galicia, german, greek, hindi, hungarian, indonesian, irish, italian, latvian, lithuanian, norwegian, persian, portuguese,romanian, russian, sorani, spanish, swedish, turkish, and thai.
+
+To use the analyzer when you map an index, specify the value within your query. For example, to map your index with the French language analyzer, enter the analyzer field and specify the `french` value for the analyzer field:
```

Comment:
The last clause sounds a bit redundant. Can it just be "...to map your index with the French language analyzer, specify the 'french' value for the analyzer field:"?